scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -929,7 +932,8 @@ Function RunTempSensorTest
         -p `"$ContainerRegistryPassword`" ``
         -t `"${ArtifactImageBuildNumber}-windows-$(GetImageArchitectureLabel)`" ``
         -tw `"$TwinTestFileArtifactFilePath`" ``
-        --optimize_for_performance=`"$OptimizeForPerformance`""
+        --optimize_for_performance=`"$OptimizeForPerformance`" ``
+        --bypass_edge_installation=`"$BypassEdgeInstallation`"
```

Comment:
is it missing a double quote at the end of this line?

Reply:
Good catch.